CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ Leaflet Changelog
 
 (all changes without author notice are by [@mourner](https://github.com/mourner))
 
+## 1.9.2 (2022-10-04)
+
+### 🐞 Bug fixes
+* Drop ESM entrypoint from package ([#8493](https://github.com/Leaflet/Leaflet/pull/8493) by [@jonkoops](https://github.com/jonkoops))
+* Check if `getElement()` is undefined before adding focus listener ([#8498](https://github.com/Leaflet/Leaflet/pull/8498) by [@Falke-Design](https://github.com/Falke-Design))
+
+### 🧪 Tests
+* Disable zoom animation for Line/PolyUtil tests ([#8478](https://github.com/Leaflet/Leaflet/pull/8478))
+
+### 📝 Docs
+* Remove `maxMargin` from `Bounds.equals` ([#8500](https://github.com/Leaflet/Leaflet/pull/8500) by [@Falke-Design](https://github.com/Falke-Design))
```

Comment:
```suggestion
* Fix docu: remove `maxMargin` from `Bounds.equals` ([#8500](https://github.com/Leaflet/Leaflet/pull/8500) by [@Falke-Design](https://github.com/Falke-Design))
```

Something like this, because we do not change the api

Reply:
Added a different description as per @mourner's suggestion.